code/modules/mob/living/living.dm

```diff
@@ -277,9 +277,14 @@
 
 		log_combat(src, M, "grabbed", addition="passive grab")
 		if(!supress_message && !(iscarbon(AM) && HAS_TRAIT(src, TRAIT_STRONG_GRABBER)))
-			M.visible_message("<span class='warning'>[src] grabs [M] [(zone_selected == "l_arm" || zone_selected == "r_arm" && ishuman(M))? "by their hands":"passively"]!</span>", \
+			if(isfelinid(AM) && isfelinid(src))
```

Comment:
you should check for cat tails instead of the felinid race.

not all felinids have (cat) tails, and people who have had cat tails surgically attached to them should be allowed to participate in the fun as well.

Reply:
absolutely ,and also check for lizard tails